ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-no-4.15.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
```

Comment:
We removed this stanza because we don't need it. If you rerun the prow script from the openshift-docs branch this will be removed.